meeting_minutes/221013_SARIF_TC_68.md

```diff
@@ -0,0 +1,154 @@
+# 1. Opening Activities
+
+## 1.1 Opening comments (Co-Chair David)
+
+## 1.2 Introduction of participants/roll call (Co-Chair David)
+
+| First Name | Last Name | Company           | Role(s)                 |
+|:-----------|:----------|:------------------|:------------------------|
+| Aditya     | Sharad    | Microsoft         | Voting Member           |
+| Charles    | Wilson    | Motional AD       | Member                  |
+| David      | Keaton    | Individual        | Chair                   |
+| Gerald     | Sullivan  | Micro Focus       | Voting Member           |
+| Johnathan  | Gilday    | Contrast Security | Voting Member           |
+| Mary       | Martin    | Microsoft         | Voting Member           |
+| Michael    | Fanning   | Microsoft         | Voting member           |
+| Nathan     | Baird     | Microsoft         | Voting Member           |
+| Paul       | Anderson  | Grammatech, Inc.  | Member                  |
+| Pavel      | Mikula    | Sonar             | Voting Member           |
+| Stefan     | Hagen     | Individual        | Secretary, taking notes |
+| Thanassis  | Avgerinos | ForAllSecure Inc  | Voting Member           |
+| Yekaterina | O'Neil    | Micro Focus       | Voting member           |
+
+## 1.3 Procedures for this meeting (Co-Chair David)
+
+## 1.4 Approval of agenda (Co-Chair David)
+
+* https://www.oasis-open.org/committees/download.php/70435/agenda_20221013.html
+
+Agenda was approved
+
+## 1.5 Approval of previous minutes (Co-Chair David)
+
+* [Minutes of 2022-09-29_Meeting #67](https://www.oasis-open.org/committees/document.php?document_id=70429&wg_abbrev=sarif)
+
+Minutes were approved
+
+## 1.6 Review of action items and resolutions (Secretary Stefan)
+
+* ACTION on David to keep pushing to close on the wikipedia page
+  * Michael continues the action
+* ACTION on Mary to continue to develop fixes scenario
+  * ONGOING
+* ACTION on Michael to provide additional references for the wikipedia
+  * COMPLETE
+* ACTION on Jeff to provide an initial seed page at github to foster the future roadmap / extension discussion
+  * ONGOING
+
+## 1.7 Identification of SARIF TC voting members (Co-Chair David)
+
+### 1.7.1 Prospective members attending their first meeting
+
+### 1.7.2 Members attaining voting rights at the end of this meeting
+
+* Charles Wilson
+* Paul Anderson
+
+### 1.7.3 Members losing voting rights if they have not joined this meeting by the time it ends
+
+### 1.7.4 Members who previously lost voting rights who are attending this meeting
+
+### 1.7.5 Members who have declared a leave of absence
```

Comment:
Shouldn't Jeff Williams be here?

Reply:
Cf. [1b7f3cd3](https://github.com/oasis-tcs/sarif-spec/pull/546/commits/1b7f3cd3768288de9effd5244a80ab6a882b8471)